multimedia/ffmpeg6/Portfile

```diff
@@ -380,17 +379,23 @@ post-destroot {
     foreach f [glob ${worksrcpath}/doc/*.txt] {
         file copy $f ${destroot}${port_docdir}
     }
+}
 
-    # Create bin symlinks
-    set port_bin_list \
-        [glob -type f -directory ${destroot}${port_bindir} *]
-    foreach f ${port_bin_list} {
-        set fname [file tail ${f}]
-        ui_info "Symlinking bin: ${prefix}/bin/${fname} -> ${port_bindir}/${fname}"
-        ln -s ${port_bindir}/${fname} ${destroot}${prefix}/bin/${fname}
+variant symlink description {Create binary symlinks for easier access} {
```

Comment:
Why make this a variant? Who would want to turn it off?

Reply:
If users follow the newly added notes to add new entry to PATH, these symlinks become redundant. What’s your suggestion?

Reply:
> Why make this a variant? Who would want to turn it off?

That was my question too, as users presently don't have to do anything to use the binaries. And AFAIK, that hasn't caused any issues.

Reply:
Could be just me but I want an easy way for ffmpeg6 to mask old ffmpeg binaries, so that I get v6 by just executing `ffmpeg`. So I did just that here. The default behavior stays the same. Adding a 6 every time is just annoying for me is all :)